fw/sock_clnt.c

```diff
@@ -935,12 +935,9 @@ tfw_cfgop_listen(TfwCfgSpec *cs, TfwCfgEntry *ce)
 static int
 tfw_cfgop_keepalive_timeout(TfwCfgSpec *cs, TfwCfgEntry *ce)
 {
-	int r;
-
-	if ((r = tfw_cfg_check_val_n(ce, 1)))
-		return -EINVAL;
+	TFW_CFG_CHECK_VAL_N(==, 1, cs, ce);
 
-	if ((r = tfw_cfg_parse_int(ce->vals[0], &tfw_cli_cfg_ka_timeout))) {
+	if (tfw_cfg_parse_int(ce->vals[0], &tfw_cli_cfg_ka_timeout)) {
```

Comment:
It's better to leave `((r = tfw_cfg_parse_int(ce->vals[0], &tfw_cli_cfg_ka_timeout)))` as is and return `r` in case of error

Reply:
Fixed